src/Data/Characters/YunJin/index.tsx

```diff
@@ -185,8 +185,6 @@ const sheet: ICharacterSheet = {
         value: datamine.burst.enerCost,
       }], {
         teamBuff: true,
-        header: conditionalHeader("burst", tr, burst),
-        description: tr("burst.description"),
```

Comment:
Why remove the header/desc here? YunJin's burst is a teambuff, which needs these fields to display, right?